pio.json5

```diff
@@ -103,17 +103,17 @@
           name: 'ODATA',
           addressOffset: 0, size: 32,
           displayName: 'Output Data Register',
-          fields: [{name: 'data', bitOffset: 0, bitWidth: 32}]
```

Comment:
Just out of curiosity, was this change necessary for correctness, or did you remove it because it was superfluous?

Reply:
necessary since the scala does not generate correctly if you use the wrong field names. `duh-scala` is following an older duh schema.